namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-incentives-dev/resources/irsa.tf

```diff
@@ -0,0 +1,36 @@
+# Add the names of the SQS & SNS which the app needs permissions to access.
+# The value of each item should be the namespace where the SQS was created.
+# This information is used to collect the IAM policies which are used by the IRSA module.
+locals {
+  sqs_queues = {
+    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-dev"
+  }
+  sns_topics = {
+    "cloud-platform-Digital-Prison-Services-e29fb030a51b3576dd645aa5e460e573" = "hmpps-domain-events-dev"
+  }
+}
+
+module "app-irsa" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=1.1.0"
+
+  eks_cluster_name = var.eks_cluster_name
+  namespace        = var.namespace
+  service_account  = "hmpps-incentives-api-${var.environment}"
```

Comment:
Included `api` in the name because both API and UI services run in the namespace so wanted to be explicit